2020/prep/selection/update_webpage.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+YEAR=2020
+DIVISIONS=../../divisions.json
+BENCHMARKS_SQ=final/benchmark_selection_single_query
+BENCHMARKS_INC=final/benchmark_selection_incremental
+BENCHMARKS_MV=final/benchmark_selection_model_valalidation
+BENCHMARKS_UC=final/benchmark_selection_unsat_core
+MAKE_DIVISION=../../../tools/selection/prepare_benchmark_ymls.py
+OUTPUT=../../../../smt-comp.github.io/_divisions_$YEAR/
+NC=non-competing
+BENCHMARKS_INC=final/benchmark_selection_incremental
+BENCHMARKS_MV=final/benchmark_selection_model_validation
+BENCHMARKS_UC=final/benchmark_selection_unsat_core
+
+
+mkdir -p non_competing
+
+../../../tools/prep/extract_data_from_solvers_divisions.py -d $DIVISIONS -c ../../registration/canon_solvers.json -n non_competing ../../registration/solvers_divisions_final.csv ~/work/smtcomp/smt-comp.github.io/_participants_$YEAR $YEAR
+
```

Comment:
Make the website path a command-line argument for the script?

Reply:
That is on purpose. I don't want to figure out the command line arguments to reproduce the results.

My rule of thumb is, the scripts in tools/* should all get the paths via command line, the scripts in 2020/* should use hard-coded paths.

Reply:
But if you meant make it also a variable in front to make the command line more readable, I can do this :)

Reply:
Wait I see, the `~/work/smtcomp` is wrong of course, it should be relative like the others paths.

Reply:
Ok... I'm just thinking that it's a different git repository, so in principle there are no guarantees that it would be accessible through the relative path.  But admittedly on my disk it's in that path as well.